kuka_rsi_driver/src/robot_manager_node_eki_rsi.cpp

```diff
@@ -67,14 +67,18 @@ RobotManagerNodeEkiRsi::RobotManagerNodeEkiRsi()
         kuka_drivers_core::ControllerType::JOINT_POSITION_CONTROLLER_TYPE, controller_name);
     });
 
-  registerParameter<int>(
+  this->registerParameter<int>(
     "control_mode", static_cast<int>(kuka_drivers_core::ControlMode::JOINT_POSITION_CONTROL),
     kuka_drivers_core::ParameterSetAccessRights{true, true},
     [this](int control_mode) { return OnControlModeChangeRequest(control_mode); });
 
-  registerStaticParameter<std::string>(
+  this->registerStaticParameter<std::string>(
```

Comment:
it should be possible to register the parameter directly as a boolean, please do that